swagger/swagger.json

```diff
@@ -197,7 +197,7 @@
     "/names/{name}": {
       "get": {
         "description": "Search for a name",
-        "operationId": "NamesSearchGet",
+        "operationId": "getName",
```

Comment:
This endpoint does not return just name, it returns a list of names. I think it has sense mention that it searching for names, not just returns records with the same name.

Reply:
updated this to `searchName`
consider this to be deployed next upcoming release.